### DISCUSSION TITLE
如何只用一个 Escape 键退出 Raycast - Get the Best of Both Worlds with Raycast and Alfred - 1nan

### DISCUSSION THREAD

**Discussion by giscus** [yinan-c/pages-comment#11] [Announcements]
# 如何只用一个 Escape 键退出 Raycast - Get the Best of Both Worlds with Raycast and Alfred - 1nan

https://yinan.me/raycast-alfred.html

**Comment by yinan-c** [comment]
I've updated the macro, to fix the bug that escape might be delayed. Here is the repository with the updated versions.
https://github.com/yinan-c/my-kmmacros
There are also macros that enable Alfred to call for Raycast AI.
